lib/token-translator.js

```diff
@@ -58,6 +58,8 @@ function convertTemplatePart(tokens, code) {
 
     if (firstToken.range) {
         token.range = [firstToken.range[0], lastTemplateToken.range[1]];
+        token.start = firstToken.range[0];
+        token.end = lastTemplateToken.range[1];
```

Comment:
nbd but maybe `token.range = [token.start, token.end]` after these lines would reduce some of the duplication?